src/model/Card.ts

```diff
@@ -3,3 +3,13 @@ export type CardProps = {
   front: string
   back: string
 }
+
+export type FlashcardProps = Omit<CardProps, 'back' | 'front'> & {
+  query: string
+  isCorrect?: boolean
+}
+
+export type FlashcardAnswerValidation = {
+  expectedAnswer: string
+  isAnswerCorrect: boolean
+}
```

Comment:
Frage: mehrere Types in einem Modul ok oder doch lieber Card.ts und separat Flashcard.ts?

Ich habe auch mal gesehen, dass man gerne Card**Types**.ts, Flashcard**Types**.ts verwendet...

Reply:
Ich fände es besser, wenn wir alle types zentral anbieten, da wir ja aufeinander aufbauen. Wäre etwas fürs cleanup am schluss 😄

Reply:
#27